weaviate/collections/grpc/query.py

```diff
@@ -599,6 +601,8 @@ def _translate_properties_from_python_to_grpc(
         if properties is None and references is None:
             return None
         return search_get_pb2.PropertiesRequest(
+            uses_new_default_logic=self._is_weaviate_version_123,
+            return_all_nonref_properties=self._is_weaviate_version_123 and properties is None,
```

Comment:
I think you can just do 
```
            uses_new_default_logic=True,
            return_all_nonref_properties=properties is None,
```

AFAIK GRPC just ignores unknown fields, so if 1.22 recieves them, they will just be ignored. But I don't really care